makelib/k8s_tools.mk

```
@@ -142,3 +152,10 @@ $(HELM3):
 	@mv $(TOOLS_HOST_DIR)/tmp-helm3/$(SAFEHOSTPLATFORM)/helm $(HELM3)
 	@rm -fr $(TOOLS_HOST_DIR)/tmp-helm3
 	@$(OK) installing helm3 $(HELM_VERSION)
+
+# kuttl download and install
+$(KUTTL):
+	@$(INFO) installing kuttl $(KUTTL_VERSION)
+	@curl -fsSLo $(KUTTL) --create-dirs https://github.com/kudobuilder/kuttl/releases/download/v$(KUTTL_VERSION)/kubectl-kuttl_$(KUTTL_VERSION)_$(HOSTOS)_$(KUTTL_DOWNLOAD_TUPLE) || $(FAIL)
```

```suggestion
	@curl -fsSLo $(KUTTL) --create-dirs https://github.com/kudobuilder/kuttl/releases/download/v$(KUTTL_VERSION)/kubectl-kuttl_$(KUTTL_VERSION)_$(HOST_PLATFORM) || $(FAIL)
```

It seems to me that `HOSTARCH` is [actually in](https://github.com/upbound/build/blob/4ecfc67/makelib/common.mk#L124) `x86_64` form. If that's the case, [`HOST_PLATFORM`](https://github.com/upbound/build/blob/4ecfc67/makelib/common.mk#L134) should work.